mzLib/Transcriptomics/Classes/NucleicAcid.cs

```diff
@@ -0,0 +1,393 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Chemistry;
+using Easy.Common.Extensions;
+using Easy.Common.Interfaces;
+using MassSpectrometry;
+using MathNet.Numerics;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace Transcriptomics
+{
+    /// <summary>
+    /// A linear polymer of Nucleic acids
+    /// </summary>
+    public abstract class NucleicAcid : INucleicAcid, IEquatable<NucleicAcid>, IHasMass
+    {
+
+        #region Static Properties
+
+        /// <summary>
+        /// The default chemical formula of the five prime (hydroxyl group)
+        /// </summary>
+        public static readonly ChemicalFormula DefaultFivePrimeTerminus = ChemicalFormula.ParseFormula("O-3P-1");
```

Comment:
is it clear what "Default" means?

Reply:
Added some comments for clarity, please let me know if that answers your question